packages/esm-patient-medications-app/src/order-basket/medication-order-form.component.tsx

```diff
@@ -332,18 +332,18 @@ export default function MedicationOrderForm({
           </Row>
           <Row style={{ marginTop: '1rem' }}>
             <Column md={2}>
-              <FormGroup legendText={t('quantityDispensed', 'Quantity Dispensed')}>
+              <FormGroup legendText={t('quantity', 'Quantity')}>
                 <NumberInput
                   light={isTablet}
-                  id="quantityDispensed"
-                  helperText={t('pillsDispensed', 'Pills dispensed')}
+                  id="quantity"
+                  helperText={t('pillstoDispense', 'Pills to dispense')}
                   value={orderBasketItem.pillsDispensed}
                   min={0}
                   onChange={(e) => {
                     setOrderBasketItem({
                       ...orderBasketItem,
                       // @ts-ignore
-                      pillsDispensed: +e.imaginaryTarget.value,
+                      pillstoDispense: +e.imaginaryTarget.value,
```

Comment:
```suggestion
                      pillsDispensed: +e.imaginaryTarget.value,
```

Code elsewhere very probably depends on this.

Reply:
Agreed - will add this as a comment. But the semantics of "pillsDispensed" is incorrect. The drug ORDER is meant to be an INSTRUCTION, not an ACTION by the doctor. It would be the Pharmacist in the Dispensing Module (which is not yet built) who we'd want to record "Dispensed" in past tense. 

Hope that makes sense, happy to discuss further.